src/godot_fmod.h

```diff
@@ -111,6 +111,8 @@ namespace godot {
         int actualListenerNumber = 0;
         Listener listeners[FMOD_MAX_LISTENERS];
         bool listenerWarning = true;
+        unsigned int bufferlength = 0;
```

Comment:
I think there is no value in storing those variables inside the class.

In the 2 getters, you can just create a local variable, pass its pointers to FMOD, then return it.

Reply:
I agree

Reply:
Will do!

Reply:
Fixed!